README.md

```diff
@@ -9,7 +9,7 @@ For further details see [Full SDK documentation](http://docs.sygictravelapi.com/
 ## Requirements
 
 - Android SDK 8.0.0 platform (O, API level 26)
-- Build tools 26.0.1
+- Build tools 26.0.2
```

Comment:
This lines should be included in appropriate commit.

Reply:
:heavy_check_mark: